components/static-pages/tos.md

```diff
@@ -38,39 +48,46 @@ You may need to register for an account to access some or all of the Platform. I
+Our Platform allows you to use and store a payment method via the Platform that is acceptable to us ("**Payment Method**") to pay for any contributions or to receive any expense payments. You represent and warrant that you are authorized to use the Payment Method. You authorize our designated third-party Payment Processor to charge the amount specified to your Payment Method for any contribution you authorize via the Platform. If the Payment Method cannot be verified, is invalid, or is not otherwise acceptable, your contribution may be suspended or cancelled automatically. Some Payment Methods may be offered as separately branded Payment Processor services (such as PayPal), and you agree to comply with the terms of those services.
+
+(c) **Recurring Payments.** You may authorize a recurring contribution to a Host for designation to a Collective by indicating as much through the Platform ("**Recurring Contribution**").
+For a Recurring Contribution, you are authorizing our designated Payment Processor to charge the amount specified to your Payment Method on the day that the initial contribution is made and to charge the same amount every calendar month thereafter (or every year if the renewal period is annual).
+If the initial contribution was made with a Platform Tip, the Platform Tip will be included on each recurring payment.
+**When you initiate a recurring contribution to a Host for designation to a Collective, you expressly agree that our Payment Processor is authorized to charge you on a monthly basis (or on an annual basis, if specified via the platform) for the same payment amount until you cancel it or we suspend or stop providing you access to the platform in accordance with these Terms.**
 
 (d) **Cancellations.** Financial Contributors can cancel or change the amount of a Recurring Contribution via [opencollective.com/recurring-contributions](https://www.opencollective.com/recurring-contributions).
```

Comment:
A small one I wanted to include in https://github.com/opencollective/opencollective-frontend/pull/11357 - update to the latest URL.

```suggestion
(d) **Cancellations.** Financial Contributors can cancel or change the amount of a Recurring Contribution via [opencollective.com/recurring-contributions](https://opencollective.com/dashboard/outgoing-contributions).
```